package/raven_framework.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
```

Comment:
I think the prescribed code style is 2-space indents. I'm open to changing it potentially for FORCE, but currently the rest of the src code is indented this way.

Reply:
I agree that it probably should be changed to 2 space indents.

Reply:
Makes sense to me. I'll change this.